### PR TITLE
modify controller & routes booking

### PR DESCRIPTION
**Pull request by MsrXvi**
-  Edit Response in findBookingById
- Add Pagination in findBookingById
- Add Validation in Update and Delete if Id not found with response
- Add middleware authenticate for all route booking

### CODE REVIEW

File: controllers/bookingController.js [hunk 1]
```diff
@@ -59,6 +59,19 @@ const createBooking = async (req, res, next) => {
 const updateBooking = async (req, res, next) => {
   const { customerId, packageId, bookingDate, numberOfPeople } = req.body;
   try {
+
+    const car = await Booking.findOne({
```
Comment (akbarrahmatm) [inline controllers/bookingController.js]: Ini ganti jadi "booking" bangg

File: controllers/bookingController.js [hunk 2]
```diff
@@ -59,6 +59,19 @@ const createBooking = async (req, res, next) => {
...
+      where: {
+        id,
+      },
+    });
+
+
+    if (!Booking) {
```
Comment (akbarrahmatm) [inline controllers/bookingController.js]: kalau yang diatas udah diganti "booking", ganti ifnya jadi "!booking"

**Review by akbarrahmatm — CHANGES_REQUESTED** [review]
Tolong diganti yaa bangg push di branch yang tadi ajaaa yahh ntar otomatis ke update kok

**Comment by MsrXvi** [comment]
> Tolong diganti yaa bangg push di branch yang tadi ajaaa yahh ntar otomatis ke update kok

diganti ke branch yang mana bang

**Comment by akbarrahmatm** [comment]
@MsrXvi langsung push aja kayak tadi bangg

**Comment by MsrXvi** [comment]
> @MsrXvi langsung push aja kayak tadi bangg

udahng bang

**Comment by akbarrahmatm** [comment]
Nice bang, udah bagus semua, approved yapss

**Comment by MsrXvi** [comment]
> Nice bang, udah bagus semua, approved yapss

Oke bang thx

**Pull request merged**